src/summaries.rs

```diff
@@ -85,18 +85,21 @@ pub struct Summary {
 /// Constructs a summary of a function body by processing state information gathered during
 /// abstract interpretation of the body.
 pub fn summarize(
-    _environment: &Environment,
+    environment: &Environment,
     _inferred_preconditions: &List<AbstractValue>,
     preconditions: &List<AbstractValue>,
     post_conditions: &List<AbstractValue>,
     unwind_condition: &List<AbstractValue>,
 ) -> Summary {
-    let result = None; // todo: #31 extract from exit environment
+    let result = environment.value_at(&Path::LocalVariable { ordinal: 0 });
     let side_effects: List<(Path, AbstractValue)> = List::new(); // todo: #31  extract from environment
     let unwind_side_effects: List<(Path, AbstractValue)> = List::new(); // todo: #31  extract from environment
     Summary {
         preconditions: preconditions.clone(),
-        result,
+        result: match result {
```

Comment:
Can you do

```
  result: v.map(|v| v.clone()),
```

Or something along those lines? Actually, can you clone result directly?